back/src/config/helpers/handleHttpJsonResponseError.ts

```diff
@@ -106,4 +107,5 @@ const logErrorAndNotifyDiscord = (
 
   logger.error(params);
   notifyObjectDiscord(params);
+  notifyObjectSlack(params);
```

Comment:
qu'est-ce que tu penses de plutôt modifier notifiyObjectDiscord, pour ne pas avoir à tout modifier partout ? un peu tard sans doute...
```suggestion
  notifyObjectToTeam(params);
```